src/Compilers/Core/Portable/Emit/SemanticEdit.cs

```diff
@@ -31,6 +31,12 @@ namespace Microsoft.CodeAnalysis.Emit
         /// </summary>
         public ISymbol? NewSymbol { get; }
 
+        /// <summary>
+        /// The symbol that matches the containing symbol of OldSymbol, but from
+        /// the later compilation. Used to turn deletes into updates.
+        /// </summary>
+        public ISymbol? NewContainingSymbol { get; }
```

Comment:
If we want to avoid the public API change, we could just use `NewSymbol` for this, as its always going to be null for a deletion anyway, but its a little dodgy